src/labeling_related.f90

```diff
@@ -712,7 +711,15 @@ SUBROUTINE write_labelings(k,n,nD,parLabel,parDigit,HNFi,HNFlist,SNFlist,L,fixOp
        ! Now convert the base-10 number (labIndx) to the correct labeling
        if(conc_check) then
           labIndx = vsL(il)
+!del          write(*,'("labIndx,concVect,labeling")')
```

Comment:
Do you want to delete the things you commented out with !del?

Reply:
Thanks Wiley, for catching these  things. I made some changes based on your feedback and a BUNCH more that are all cosmetic, like removing print statements, old comments, adding new comments that made it easier when debugging this time.

Take a look now and see what you think. I think my changes from this morning have been added to the PR.